import-scripts/clear_cbioportal_persistence_cache.sh

```diff
@@ -23,52 +18,19 @@ portal_to_deployment_map["triage"]="eks-triage"
-portal_to_deployment_map["prostate"]="eks-prostate"
 portal_to_deployment_map["sclc"]="eks-sclc"
-portal_to_deployment_map["target"]="eks-target"
-portal_to_deployment_map["triage"]="eks-triage"
 
-unset portal_to_url
-declare -A portal_to_url
-portal_to_url["public"]="https://cbioportal.org"
-portal_to_url["genie-public"]="https://genie.cbioportal.org"
-portal_to_url["genie-private"]="https://genie-private.cbioportal.org"
-portal_to_url["genie-archive"]="https://genie-archive.cbioportal.org"
-portal_to_url["triage"]="https://triage.cbioportal.mskcc.org"
-portal_to_url["msk"]="https://cbioportal.mskcc.org"
-portal_to_url["msk-beta"]="https://msk-beta.cbioportal.mskcc.org"
-portal_to_url["private"]="https://private.cbioportal.mskcc.org"
-portal_to_url["su2c"]="https://su2c.cbioportal.mskcc.org"
-portal_to_url["acc"]="https://acc.cbioportal.mskcc.org"
-portal_to_url["glioma"]="https://glioma.cbioportal.mskcc.org"
-portal_to_url["immunotherapy"]="https://immunotherapy.cbioportal.mskcc.org"
-portal_to_url["kras"]="https://kras.cbioportal.mskcc.org"
-portal_to_url["pdx"]="https://pdx.cbioportal.mskcc.org"
-portal_to_url["poetic"]="https://poetic.cbioportal.mskcc.org"
-portal_to_url["prostate"]="https://prostate.cbioportal.mskcc.org"
-portal_to_url["sclc"]="https://sclc.cbioportal.mskcc.org"
-portal_to_url["target"]="https://target.cbioportal.mskcc.org"
-#TODO : delete these overrides .. these are in place for testing until our portals are made publicly visible
-portal_to_url["msk"]="https://msk.cbioportal.aws.mskcc.org"
-portal_to_url["msk-beta"]="https://beta.cbioportal.mskcc.org"
-portal_to_url["private"]="https://private.cbioportal.aws.mskcc.org"
-portal_to_url["su2c"]="https://su2c.cbioportal.aws.mskcc.org"
-portal_to_url["acc"]="https://acc.cbioportal.aws.mskcc.org"
-portal_to_url["glioma"]="https://glioma.cbioportal.aws.mskcc.org"
-portal_to_url["immunotherapy"]="https://immunotherapy.cbioportal.aws.mskcc.org"
-portal_to_url["kras"]="https://kras.cbioportal.aws.mskcc.org"
-portal_to_url["pdx"]="https://pdx.cbioportal.aws.mskcc.org"
-portal_to_url["poetic"]="https://poetic.cbioportal.aws.mskcc.org"
-portal_to_url["prostate"]="https://prostate.cbioportal.aws.mskcc.org"
-portal_to_url["sclc"]="https://sclc.cbioportal.aws.mskcc.org"
-portal_to_url["target"]="https://target.cbioportal.aws.mskcc.org"
+unset portal_to_cache_service_list
+declare -A portal_to_cache_service_list
+portal_to_cache_service_list["public"]="cbioportal-public-persistence-redis-master cbioportal-public-persistence-redis-slave"
+portal_to_cache_service_list["genie-public"]="cbioportal-persistence-redis-genie-master cbioportal-persistence-redis-genie-slave"
+portal_to_cache_service_list["genie-private"]="cbioportal-persistence-redis-genie-master cbioportal-persistence-redis-genie-slave"
+portal_to_cache_service_list["genie-archive"]=""
+portal_to_cache_service_list["triage"]="cbioportal-persistence-redis-master cbioportal-persistence-redis-replicas"
```

Comment:
triage will be it's own I believe

Reply:
Adjusting now. There is now a running triage-cbioportal-persistence-redis helm installation and triage is using it.